src/index.spec.ts

```diff
@@ -1,12 +1,176 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+let noInput: any;
+const emptyString: string = '';
+const emptyArray: any = [];
+const emptyObject: any = {};
+const randomString: any = 'halsfhals';
+const randomNumber: any = 123131;
+const invalidProduct = {
+  name: 'Raw legums',
+  tags: ['vegetable', 'dessert'],
+  description: 'Raw legums on the wooden table',
+  price: 17.11,
+};
+const validProduct = {
+  name: 'Asparagus',
+  tags: ['vegetable'],
+  description: 'Asparagus with ham',
+  price: 18.95,
+};
+
+describe('Takes a number or string and returns a boolean on determining if is integer', () => {
```

Comment:
We usually use the tested function name as text in the describe operator. So, for this case:
```ts
describe('isInteger',()=>{
  // ...your test cases here
})
```

The same apply for the other `describe()`.

Reply:
Got it! Thanks!